src/UserWarning.tsx

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
 
-export const UserWarning: React.FC = () => (
-  <section className="section">
-    <p className="box is-size-3">
-      Please get your
-      {' '}
-      <b> userId </b>
-      {' '}
-      <a href="https://mate-academy.github.io/react_student-registration">
-        here
-      </a>
-      {' '}
-      and save it in the app
-      {' '}
-      <pre>const USER_ID = ...</pre>
+export const UserWarning: React.FC = () => {
```

Comment:
U dont have to import React. Instead of 'React.FC' u can just put 'FC'

Reply:
done